build.sh

```diff
@@ -40,39 +40,43 @@ do
     test)
       # run lang-specific tests
       (cd lang/java; ./build.sh test)
+
+      # create interop test data
+      mkdir -p build/interop/data
+      (cd lang/java/avro; mvn -B -P interop-data-generate generate-resources)
```

Comment:
Shouldn't these be written as `cd ... &&` rather than `cd ...;`?

Reply:
Good point